sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/models/SqlRuleFilter.java

```diff
@@ -99,4 +100,46 @@ Boolean isPreprocessingRequired() {
     public String getSqlExpression() {
         return sqlExpression;
     }
+
+    /**
+     * Converts the value of the current instance to its equivalent string representation.
+     *
+     * @return A string representation of the current instance.
+     */
+    @Override
+    public String toString() {
+        return String.format("SqlRuleFilter: %s", sqlExpression);
+    }
+
+    /**
+     *  Compares this RuleFilter to the specified object. The result is true if and only if the argument is not null
+     *  and is a SqlRuleFilter object that with the same parameters as this object.
+     *
+     * @param other - the object to which the current SqlRuleFilter should be compared.
+     * @return True, if the passed object is a SqlRuleFilter with the same parameter values, False otherwise.
+     */
+    @Override
+    public boolean equals(Object other) {
```

Comment:
Missing null check on `other`

Reply:
instanceof operator should be covering the null check as well